src/pages/index.tsx

```diff
@@ -48,6 +49,10 @@ const modeDispatchPages: Record<AppMode, () => React.ReactElement> = {
           <Route path="/workspace/:uuid">
             <WorkspaceMission />
           </Route>
+          <Route path="/workspaces/:uuid">
+            <AddRepos />
```

Comment:
You do not need to create a new route for repositories, the repositories are to be in this file according to the design 
```sh
 <Route path="/workspace/:uuid">
       <WorkspaceMission />
 </Route>
```

Reply:
okay will move it to that